.github/workflows/staging-tests.yml

```diff
@@ -0,0 +1,45 @@
+name: Staging Instance Tests
+
+permissions:
+  # Needed to access the workflow's OIDC identity.
+  id-token: write
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+  schedule:
+    - cron: '0 12 * * *'
```

Comment:
Kind of inclined to make this more frequent, thoughts?

Reply:
Makes sense to me. Every 4/6 hours maybe?

Reply:
Set to 4 hours.